lib/maps/thunderdome.txt

```diff
@@ -0,0 +1,12 @@
+HW|  |  |  |  |  |  |  |  |  |  |  
+  |  |  |  |  |  |  |  |  |  |  |  
+  |IM|IM|IM|IM|IM|IM|IM|  |IM|  |  
+  |IM|  |  |  |  |  |  |  |IM|  |  
+  |IM|  |  |  |  |  |  |  |IM|  |
+  |IM|  |  |  |DM|  |  |  |IM|  |  
+  |IM|  |  |  |  |  |  |  |IM|  |  
+  |IM|  |  |  |  |  |  |  |IM|  |  
+  |IM|  |IM|IM|IM|IM|IM|IM|IM|  |  
+  |  |  |  |  |  |  |  |  |  |  |  
+  |  |  |  |  |  |  |  |  |  |  |  
+  |  |  |  |  |  |  |  |  |  |HW|  
```

Comment:
You can make the map a little more symmetrical by moving the `HW` up and placing a wall at the bottom of the map. Currently, all maps must be 12x12 until we make adjustments to the engine and probably the website.

I'll leave it up to you, just let me know if you want to tweak this one or not, then I'll merge it! Thanks for the PR.